### DISCUSSION TITLE
Disabling the "update available" toast

### DISCUSSION THREAD

**Discussion by acf** [cotes2020/jekyll-theme-chirpy#2078] [Q&A] [answered]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

Hi there,  I'm just getting started with Jekyll (and chirpy) so apologies if this is a silly question.

I find that when I push a change and then refresh the page in safari, I get the "update available" notification toast (which seems sensible).  When I press the button, the page definitely reloads but relaunches the toast and I end up in a weird loop.  I have to close the browser, or empty caches, to get the changes to appear.

Honestly, I'm not at all bothered by not having up to the minute updates so would really prefer just to disable the notification as it doesn't seem to be working for me.  Is it possible to do this?  I'm relatively comfortable with the yaml, the ruby and the liquid, but this seems to be down in the JS and I'm not sure I want to go wading in there... I'm hoping there's a config variable I can set :-)

Any help appreciated!

Alan

### Operations you have already tried

I figured out where the notification is happening but really dont want to mess with the JS.

https://github.com/cotes2020/jekyll-theme-chirpy/blob/a16aa7d41e3c3cb28649bfa1361e8bcb91b9ca47/_javascript/pwa/app.js#L4-L6

Is that checking for a config variable called "register" ?

**Comment by cotes2020** [comment, marked as answer]
There is an option to disable the PWA feature:

https://github.com/cotes2020/jekyll-theme-chirpy/blob/a16aa7d41e3c3cb28649bfa1361e8bcb91b9ca47/_config.yml#L140-L143

**Reply by acf** [reply]
Thanks!  I didn't understand what PWA was.

**Reply by rafisics** [reply]
I like to do the opposite. I am not sure why it got disabled! 
https://github.com/cotes2020/jekyll-theme-chirpy/discussions/2258#discussion-7977990
